client/my-sites/marketing/tools/index.tsx

```diff
@@ -119,9 +119,9 @@ export const MarketingTools: FunctionComponent = () => {
 				<MarketingToolsFeature
 					title={ translate( 'Want to build a great brand? Start with a great logo' ) }
 					description={ translate(
-						'A custom logo helps your brand pop and makes your site memorable. Make a professional logo in a few clicks with our partner Fiverr.'
+						'A custom logo helps your brand pop and makes your site memorable. Make a professional logo in a few clicks with our partner Logotron.'
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.